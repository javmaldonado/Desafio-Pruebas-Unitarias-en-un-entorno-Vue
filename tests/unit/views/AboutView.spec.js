import { mount } from "@vue/test-utils";
import AboutView from "@/views/AboutView.vue";

describe("Tests de vista About", () => {
    it("Test1 - El snapshot de la vista About crea Correctamente", () => {
      const wrapper = mount(AboutView);  // Seleccionar el componente que será puesto a prueba //
      expect(wrapper.html()).toMatchSnapshot(); // Realiza una afirmación sobre la prueba si  hay Match pasa la prueba //
    });
  
  });
  