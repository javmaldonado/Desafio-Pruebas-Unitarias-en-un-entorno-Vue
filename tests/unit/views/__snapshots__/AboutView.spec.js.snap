// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests de vista About Test1 - El snapshot de la vista About crea Correctamente 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
